staff/robert-anthony/precurso/master/styles/styles.scss

```diff
@@ -0,0 +1,308 @@
+/*
```

Comment:
DRY: styles/styles.scss is redundant, why not styles/main.scss instead. in fact "main" is more accurate to name your principal style than just the repetitive "styles.scss"

Reply:
Tienes razón....